models/Card.js

```diff
@@ -40,9 +40,14 @@ Card.init(
       type: DataTypes.INTEGER,
       allowNull: false,
     },
-    e_factor: {
+    efactor: {
       type: DataTypes.FLOAT,
       allowNull: false,
+      field: 'e_factor',
```

Comment:
Instead of linking it to the e_factor field in the db I think it for consistency we name them the same across the code and db. So if it's `efactor` here it should be `efactor` in the db.

Reply:
That is a good point. Sorry to make things confusing... That wasn't my intention!

Reply:
You are right, we should keep it consistent.